tests/api/id/[form]/bearer.test.js

```diff
@@ -4,6 +4,7 @@ import retrieve from "../../../../pages/api/id/[form]/bearer";
 import executeQuery from "../../../../lib/integration/queryManager";
 import jwt from "jsonwebtoken";
 import { logMessage } from "../../../../lib/logger";
+import { checkLogs } from "../../../../lib/jestUtils";
```

Comment:
It would be cleaner if it was `@lib/jestUtils`. I believe the issue that required all relative paths was fixed a while ago.

Reply:
Thanks James, I'll go through all the tests and refactor since I'm in there.